addons/addon-base-raas/packages/base-raas-services/lib/environment/environment-mount-service.js

```diff
@@ -667,16 +667,53 @@ class EnvironmentMountService extends Service {
     return policyDoc.Statement;
   }
 
+  async _getPolicyDoc(possiblePolicyNames, roleName, workspaceRoleArn, iamClient) {
```

Comment:
nit: would suggest adding JSDocs to this method since this method some non-obvious work, something like
```
This method looks for inline policy based on the given array of "possiblePolicyNames". The method returns as soon as it finds an inline policy in the given role (identified by the "roleName") with a matching name from the "possiblePolicyNames". If no inline policy is found with any of the names from the "possiblePolicyNames", the method returns undefined. 
```

Reply:
nit: just realized `possiblePolicyNames` is more like  `possiblePolicyNameSuffixes`

Reply:
Sounds good. Done